templates/controllers/grid/issues/form/issueForm.tpl

```diff
@@ -43,16 +43,15 @@
 		{assign var=issuePublished value=false}
 	{/if}
 
-	{if $issuePublished}
 		{fbvFormArea id="datePublishedArea" title="editor.issues.datePublished"}
 			{fbvFormSection}
 				{if $issuePublished}
 					{fbvElement type="text" id="datePublished" value=$datePublished size=$fbvStyles.size.SMALL class="datepicker"}
+				{else}
+					{fbvElement type="text" id="datePublished" value=$datePublished size=$fbvStyles.size.SMALL class="datepicker" label="editor.issues.datePublished.notPublished.description"}
```

Comment:
Here we are adding the same `fbvElement` with the addition of the `label`. Would it be better to use the `if` clause to define a `label` string to `""` or `"editor.issues.datePublished.notPublished.description"` and add the `fbvElement` once?

Or to refrain the wording of the label so we can add that changed label to both cases?